contrib/epee/src/CMakeLists.txt

```diff
@@ -26,11 +26,19 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 # THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+set(EPEE_INCLUDE_DIR_BASE "${CMAKE_CURRENT_SOURCE_DIR}/../include")
+
+file(GLOB EPEE_HEADERS_PUBLIC 
+	"${EPEE_INCLUDE_DIR_BASE}/*.h*"
+	"${EPEE_INCLUDE_DIR_BASE}/**/*.h*"
```

Comment:
h* to be able to find the hpp files as well, if such are added.

Reply:
Why not `{h,hpp}` instead of `h*`?

Reply:
I've just checked it. It doesn't work.